.github/workflows/release.yml

```diff
@@ -13,254 +12,99 @@ permissions:
 
 
 jobs:
-  pypi-deploy:
-    if: "!github.event.release.prerelease"
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.9'
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install setuptools wheel twine
-      - name: Build and publish
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
-        run: |
-          python setup.py sdist bdist_wheel
-          twine upload dist/*
 
-  deb-package:
+  describe:
+    name: Version ${{ github.ref_name }}
     runs-on: ubuntu-latest
     steps:
-      - name: Install Makedeb
-        run: |
-          wget -qO - 'https://proget.hunterwittenborn.com/debian-feeds/makedeb.pub' | gpg --dearmor | sudo tee /usr/share/keyrings/makedeb-archive-keyring.gpg &> /dev/null
-          echo 'deb [signed-by=/usr/share/keyrings/makedeb-archive-keyring.gpg arch=all] https://proget.hunterwittenborn.com/ makedeb main' | sudo tee /etc/apt/sources.list.d/makedeb.list
-          sudo apt update
-          sudo apt install makedeb
-      - name: Prepare source directory
-        run: |
-          git clone https://mpr.makedeb.org/rare build
-          sed -i "s/pkgver=.*/pkgver=${{ github.event.release.tag_name }}/g" build/PKGBUILD
+      - shell: bash
+        run: "true"
```

Comment:
`bash` already is the default for Ubuntu: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell.